### DISCUSSION TITLE
Redmi Smart Band Pro

### DISCUSSION THREAD

**Discussion by marciomalcher** [NightscoutFoundation/xDrip#2113] [Ideas]
Include support for the smart band Redmi Smart Band Pro. MODEL M2101B1.

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
This might happen in the future when this work will resume https://github.com/NightscoutFoundation/xDrip/pull/2004 but someone needs to work on this smartband integration if the ususal freemyband trick can apply to it.
